core/src/main/kotlin/io/github/thanosfisherman/gasket/Gasket.kt

```diff
@@ -142,7 +117,7 @@ class Gasket(viewport: Viewport) {
         }
     }
 
-    fun draw() {
+    fun draw(shape: ShapeRenderer) {
 
         val delta: Long = TimeUtils.millis() - lastTimeCounted
```

Comment:
How would you go about refactoring this to use Gdx.graphics.deltatime as per your recommendation? What I do here is change the sinceChange condition here like so `if (sinceChange >= 1000)` and now I get a new circle every second which is what I want for debuging purposes. How could we use the Gdx delta time for that?